Assets/Scripts/Models/Inventory.cs

```diff
@@ -191,4 +159,13 @@ private void ImportPrototypeSettings(int defaulMaxStackSize, float defaultBasePr
             Category = defaultCategory;
         }
     }
+
+    private void OnStackSizeChanged(Inventory inventory)
```

Comment:
I would recond to "read" this method as - This is the place where all On.... things would happen.

But if someone more think that this is not good - I will change it to `InvokeStackSizeChanged`